lib/ansible/modules/network/netscaler/netscaler_nitro_request.py

```diff
@@ -0,0 +1,899 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#  Copyright (c) 2017 Citrix Systems
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: netscaler_nitro_request
+short_description: Issue NITRO API requests to a Netscaler instance.
+description:
+    - Issue NITRO API requests to a Netscaler instance.
+    - This is intended to be a short hand for using the uri Ansible module to issue the raw HTTP requests directly.
+    - It provides consistent return values and has no other dependencies apart from the base Ansible runtime environment.
+    - This module is intended to run either on the ansible  control node or a bastion (jumpserver) with access to the actual netscaler instance
+
+
+version_added: "2.5.0"
+
+author: George Nikolopoulos (@giorgos-nikolopoulos)
+
+options:
+
+    nsip:
+        description:
+            - The ip address of the netscaler or MAS instance where the nitro API calls will be made.
```

Comment:
`IP`
`Netscaler` - Applies to a number of places
`Nitro` - Applies to a number of places